apps/cowswap-frontend/src/legacy/state/orders/hooks.ts

```diff
@@ -223,29 +228,23 @@ export const useCombinedPendingOrders = ({
         creating: PartialOrdersMap
       }
     | undefined
-  >((state) => {
-    const ordersState = chainId && state.orders?.[chainId]
-    if (!ordersState) {
-      return
-    }
+  >((state) => state.orders?.[chainId])
```

Comment:
```
Selector unknown returned a different result when called with the same parameters. This can lead to unnecessary rerenders.
Selectors that return a new reference (such as an object or an array) should be memoized: https://redux.js.org/usage/deriving-data-selectors#optimizing-selectors-with-memoization 
```